src/browser/new-browser.ts

```diff
@@ -133,15 +159,18 @@ export class NewBrowser extends Browser {
         return options as RemoteOptions;
     }
 
-    protected _extendCapabilities(config: BrowserConfig): WebdriverIO.Capabilities {
+    protected async _extendCapabilities(config: BrowserConfig): Promise<WebdriverIO.Capabilities> {
         const capabilitiesExtendedByVersion = this.version
             ? this._extendCapabilitiesByVersion()
             : config.desiredCapabilities;
         const capabilitiesWithAddedHeadless = this._addHeadlessCapability(
             config.headless,
             capabilitiesExtendedByVersion!,
         );
-        return capabilitiesWithAddedHeadless;
+
+        return this._isLocalGridUrl()
+            ? this._addExecutablePath(config, capabilitiesWithAddedHeadless)
```

Comment:
Here we don't check for automationProtocol, so we could use our binaries for "devtools" automation protocol